packages/ui/src/components/va-tree-view/hooks/useTreeView.ts

```diff
@@ -90,8 +51,8 @@ const useTreeView: UseTreeViewFunc = (props, emit) => {
     getNodeProperty,
   } = useTreeHelpers(props)
   const { nodes, expandAll, filter, filterMethod, textBy } = toRefs(props)
-  const expandedList = useNamedStateful<TypeModelValue>('expanded', props, emit, [])
-  const checkedList = useNamedStateful<TypeModelValue>('checked', props, emit, [])
+  const { valueComputed: expandedList } = useStateful<TypeModelValue>(props, emit, undefined, 'expanded')
```

Comment:
I sugges we remove `defaultValue` in context of this PR.

You can create a new method for tree view, then deprecate existing method and plan full transition in another issue if it feels too complicated.